bin/mv.js

```diff
@@ -0,0 +1,280 @@
+}
+
+function moveWebtask(profile, name, target, options) {
+    debug('moveWebtask: profile=%j, name=%s, target=%j', _.omit(profile, 'token'), name, target);
+
+    let sourceWebtask;
+    let sourceParams = {
+        name: name,
+        container: profile.container,
+        profile: profile.name,
+        url: profile.url,
+        token: profile.url
+    };
+
+    if (equal(sourceParams, target)) {
+        throw Cli.error.invalid('Webtasks are identical. Use a different target name, container or profile.');
+    }
+
+    return profile.getWebtask({name: name})
+        .catch(function (err) {
+            if (err.statusCode === 404) {
+                throw Cli.error.notFound('No such webtask: ' + chalk.bold(name));
+            }
+            throw err;
+        })
+        .then(function (webtask) {
+            sourceWebtask = webtask;
+            return webtask.inspect({
+                decrypt: true,
+                fetch_code: true
+            });
+        })
+        .then(function (data) {
+            return copy(sourceWebtask, data, target, options);
+        })
+        .then(function () {
+            if (options.noDelete) return;
+            debug('remove: webtask=%j', sourceWebtask);
+            return sourceWebtask.remove();
+        });
+}
+
+function equal(sourceParams, targetParams) {
+    return _.isEqual(sourceParams, {
+        name: targetParams.name,
+        container: targetParams.container || sourceParams.container,
+        profile: targetParams.profile || sourceParams.profile,
+        url: targetParams.url || sourceParams.url,
+        token: targetParams.token || sourceParams.token
+    });
+}
+
+function copy(webtask, data, target, options) {
+    debug('copy: webtask=%j, data=%j, target=%j', _.omit(webtask, 'token'), data, target);
+    if (!data.jtn) throw Cli.error.cancelled('Not a named webtask.');
+
+    return coroutine(function*() {
+        target.profile = loadProfile(target.profile, target.url, target.container, target.token) || webtask.sandbox;
+        target.container = target.container || target.profile.container;
+        debug('copy: target.profile=%j', target.profile);
+
+        let claims = cloneWebtaskData(data);
+        claims.jtn = target.name || claims.jtn;
+        claims.ten = target.container || claims.ten;
+
+        try {
+            yield target.profile.createRaw(claims);
+            yield moveCronJob(webtask.sandbox, data.jtn, target, _.merge({verify: webtask.token}, options));
+            yield copyStorage(webtask, target);
+        } catch (err) {
+            throw Cli.error.cancelled('Failed to create webtask. ' + err);
+        }
+    })();
+}
+
+function loadProfile(name, url, container, token) {
+    debug('loadProfile: name=%s, url=%j, container=%j, token=%j', name, url, container, token);
+
+    let profile;
+    let config;
+
+    if (name) {
+        config = new ConfigFile();
+        profile = config.getProfile(name);
+    }
+
+    if (url || token) {
```

Comment:
Is overwritten one step later: https://github.com/callemo/wt-cli/blob/4cb608fd39fa807dff961e87260e395e8ce4a9ca/bin/mv.js#L158-L158